<launch>

  <node name="msg_client" pkg="rct_turtlebot" type="msg_client"  output="screen">
    <param name="ros_master_uri"  value="http://157.82.4.78:11311" />
    <param name="rate"  value="50.0" />
    <param name="topic_name"  value="/robot/pose_pub" />
    <param name="topic_type"  value="geometry_msgs/PoseStamped" />

  </node>

</launch>
